docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -223,3 +223,27 @@ If you plan on connecting more than 10,000 nodes or agents, please contact your
 
 Teleport components communicate with themselves using mTLS, with a separate certificate authority for each tenant. Connections to AWS services, such as DynamoDB and
 S3, are established using encryption provided by AWS, both at rest and in transit. Each tenant has its own credentials that isolate it to interacting with only its own data.
+
+### How can I find version information on connected agents?
+
+To find the version of Teleport agent is attached to your cluster, run the
+following commands for the different protocols we support.
+
+```code
+$ tctl get nodes --format=text
+$ tctl get kube_service --format=text
```

Comment:
@zmb3 I'm being told this should be `kube_servers`, and that it's obsolete and just there to support v11 clients.